src/jl_uv.c

```diff
@@ -427,10 +427,7 @@ JL_DLLEXPORT void jl_uv_writecb(uv_write_t *req, int status)
     }
 }
 
-// Note: jl_write() is called only by jl_vprintf().
-// See: doc/devdocs/stdio.rst
```

Comment:
it's really odd that we documented a `static` function, but apparently we did. Updated that with the new name